debian/changelog

```diff
@@ -1,5 +1,11 @@
+tarantool-migrate (0.0.19-1) unstable; urgency=low
+
+  * Reviving the module for tarantool 1.10+
+
+ -- Leonid Vasiliev <lvasiliev@tarantool.org>  Mon, 21 Sep 2020 11:30:00 +0300
+
 tarantool-migrate (0.0.1-1) unstable; urgency=low
 
   * Initial release
 
- -- Eugene Blikh <bigbes@tarantool.org>  Mon, 29 Feb 2016 11:30:00 +0300
+ -- Eugene Blikh <bigbes@tarantool.org>  Mon, 29 Feb 2016 11:30:00 +0300
```

Comment:
Nit: No newline symbol at end of the file.

Reply:
Fixed.